srcpkgs/deluge/template

```diff
@@ -1,20 +1,21 @@
 # Template file for 'deluge'
 pkgname=deluge
-version=2.0.3
-revision=15
+version=2.0.5
+revision=1
 build_style=python3-module
```

Comment:
add quotes just for consistency

Reply:
I have no idea what I was thinking when I wrote this, it's the other way around, we have only 12 python3 packages with quotes and 1000 without. Sorry for the confusion.